src/main/java/core/basesyntax/service/FileWriter.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax.service;
+
+public interface FileWriter {
+    void writerDataToFile(String fileName, String report);
+}
```

Comment:
```suggestion
public interface FileWriter {
    void writerDataToFile(String report, String fileName);
}
```
check it. I see different order in interface and its implementation